src/Miunie.Discord/Embeds/EmbedConstructor.cs

```diff
@@ -26,13 +27,31 @@ namespace Miunie.Discord.Embeds
     internal static class EmbedConstructor
     {
         private static readonly int RepLogPageSize = 10;
+        private static readonly uint DefaultEmbedColor = 0xEC407A; // 236, 64, 122
```

Comment:
```suggestion
        private static readonly uint DefaultEmbedColor = 0xEC407A;
```

It's not implicitly understandable what color that is by the comment. So might as well not have it. 😅

Reply:
I mean, I understand it's most likely the Miunie-pink, but I don't think it matters as much to warrant an explanation.

Reply:
Would you want to rename the color to `MiuniePinkColor` instead? It would explicitly specify the kind of color it is.

Reply:
Sure 😊